run-error-prone.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
This script shouldn't be needed anymore since we got the plugin to work instead. Let's delete this.